### PR TITLE
docs: Docs

### PR DESCRIPTION
**Pull request by Gadgetoid**
Docs!

image

### CODE REVIEW

**Comment by Gadgetoid** [comment]
cursed girlfriend meme

image

File: README.md [hunk 1]
```diff
@@ -1,19 +1,41 @@
-# Pimoroni PGA Boilerplate
+# Explorer
```
Comment (ZodiusInfuser) [inline README.md:1]: Pimoroni Explorer?

File: README.md [hunk 2]
```diff
@@ -1,19 +1,41 @@
...
 
-This repository is intended to provide a baseline MicroPython build for PGA2040
-and PGA2350, in addition to being a minimal example of how you might set up your
-own custom MicroPython flavour to support your PGA-based project.
+This repository is home to the Explorer MicroPython fimrware and examples.
```
Comment (ZodiusInfuser) [inline README.md:3]: firmware

File: README.md [hunk 3]
```diff
@@ -1,19 +1,41 @@
...
+# Explorer
 
-This repository is intended to provide a baseline MicroPython build for PGA2040
-and PGA2350, in addition to being a minimal example of how you might set up your
-own custom MicroPython flavour to support your PGA-based project.
+This repository is home to the Explorer MicroPython fimrware and examples. 
 
-Use this repository as a boilerplate to set up your own project, and GitHub actions
-should automatically handle building MicroPython for you.
+Explorer is a electronic adventure playground for physical computing, built around the RP2350B. Includes a 2.8" LCD screen, a speaker, a mini breadboard, and much more!
```
Comment (ZodiusInfuser) [inline README.md:5]: It includes

File: README.md [hunk 4]
```diff
@@ -1,19 +1,41 @@
...
+3. Drag and drop your chosen `.uf2` file onto the `RP2350` drive that appears.
+4. Your board should reset and, if you used the `with-filesystemz build, should dipslay a menu of examples.
+
+# Examples
+
+We've tried to cover all the bases with some simple examples to get you started. See [examples/README.md](examples/README.md) for more information.
+
+# Documentation
+
+To help you get started we've created a function refernece. See [docs/reference.md](docs/reference.md)
```
Comment (ZodiusInfuser) [inline README.md:41]: reference

**Pull request merged**